packages/governance/src/paramGovernance/paramManager.js

```diff
@@ -271,6 +271,8 @@ const makeParamManagerBuilder = zoe => {
     return harden(descriptions);
   };
 
+  // XXX some type safety but you don't know what keys are valid
```

Comment:
What does `XXX` mean? Is it something you meant to address before taking the PR out of DRAFT?

Reply:
I've used `XXX` to note tech debt. The idea is to call out a decision was made to let this slide but reader please don't emulate it.

What's a good way to express this in this codebase?

Reply:
Maybe `XXX` is common already?
https://github.com/Agoric/agoric-sdk/blob/b492829b8810ec7faf4f03b3d6ccff3e4c856387/packages/SwingSet/src/kernel/virtualObjectManager.js#L721
https://github.com/Agoric/agoric-sdk/blob/b492829b8810ec7faf4f03b3d6ccff3e4c856387/packages/store/src/patterns/patternMatchers.js#L400

Reply:
> I've used `XXX` to note tech debt. The idea is to call out a decision was made to let this slide but reader please don't emulate it.

Hm. That seems reasonable.

> What's a good way to express this in this codebase?

Add a note in CONTRIBUTING.md? or in something near https://github.com/Agoric/agoric-sdk/wiki/Coding-Style ?

Reply:
FWIW: I use `@@` for "do not merge to master!" (originally, it was "make sure this doesn't compile; I have to run to a meeting.")

And I use `TODO:` and `ISSUE:` without a lot of consistency about when I intend to resolve them.

Reply:
You got it: https://github.com/Agoric/agoric-sdk/wiki/Coding-Style#code-comments